### DISCUSSION TITLE
how to add dynamic styles based on trait flag?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4476] [Q&A]
Hi @artf ...
Thank ypu for your amazing grapesjs.
how to add style sector properties based on trait for example if i add checkbox in trait if its checked i need to add style properties like repeatwidth and repeatheight... when uncheck it should hide..
can you tell me how to do that in our grapesjs?
screenshot for refrence..stylemanager
![image](https://user-images.githubusercontent.com/50009624/180745548-e314298d-89e9-40d1-87f7-5c0e70ed777f.png)

**Comment by senthil177cse** [comment]
Hello @artf  please update ASAP..

**Comment by ronaldohoch** [comment]
Hello, i think this can help you.

https://github.com/artf/grapesjs/discussions/4345
